app/src/main/java/nl/eduvpn/app/service/EduOpenVPNService.java

```diff
@@ -454,11 +461,9 @@ public void detachConnectionInfoListener() {
         VpnStatus.removeByteCountListener(_byteCountListener);
     }
 
-    public interface ConnectionInfoCallback {
-        void updateStatus(Long secondsConnected, Long bytesIn, Long bytesOut);
-
-        void metadataAvailable(String localIpV4, String localIpV6);
+    @NotNull
+    public String getAbout() {
```

Comment:
I think `protocolName` would be more fitting

Reply:
Fixed in https://github.com/eduvpn/android/pull/318/commits/3daf51e35b382e1b52c8e155a57f1fd45e54e201